rskj-core/src/test/java/co/rsk/rpc/netty/Web3WebSocketServerTest.java

```diff
@@ -46,14 +48,14 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
 
 public class Web3WebSocketServerTest {
 
     private static JsonNodeFactory JSON_NODE_FACTORY = JsonNodeFactory.instance;
     private static ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final int DEFAULT_WRITE_TIMEOUT = 30;
```

Comment:
maybe the same 'seconds' comment from before applies here as well

Reply:
renamed